apps/meteor/client/components/ImageGallery/ImageGallery.tsx

```diff
@@ -176,19 +176,19 @@ export const ImageGallery = ({ images, onClose, loadMore }: { images: IUpload[];
 						modules={[Navigation, Zoom, Keyboard, A11y]}
 						onInit={(swiper) => setSwiperInst(swiper)}
 						onSlidesGridLengthChange={(swiper) => {
-							swiper.slideTo(images.length - gridSize, 2000);
+							swiper.slideTo(images.length - gridSize);
```

Comment:
This is not really causing the issue, but I found out the effect just looks better when using the default speed. I'm not even sure if it's worth a new regression, so that's why I decided to include it here :thinking: do you think it should be in another PR?

Reply:
Yup, better to separate into another task with its own task

Reply:
Here it is: #32581